### PR TITLE
Feature/product description

### PR DESCRIPTION
**Pull request by Akashakashpal**

### CODE REVIEW

File: src/components/LocationModal/LocationModal.tsx [hunk 1]
```diff
@@ -58,11 +70,11 @@ function LocationModal({ isOpen, onClose }: IPropsLocationModal): JSX.Element {
                   placeholder="Enter an address or ZIP code"
                   sx={{
```
Comment (dewale005) [inline src/components/LocationModal/LocationModal.tsx:71]: you need to take the style out of the element

**Review by dewale005 — CHANGES_REQUESTED** [review]
Add the interfaces for this components

**Comment by Akashakashpal** [comment]
working on it

File: src/Layout/ProductHeader/ProductsHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  return (
+    <Flex
+      as="nav"
+      align="center"
+      justify="space-between"
+      wrap="wrap"
+      paddingX={{ base: '1rem', lg: '4rem' }}
+      height="6rem"
+      color="white"
+      backgroundColor={{ base: 'FFFFFF', lg: '#F7F8F9' }}
```
Comment (dewale005) [inline src/Layout/ProductHeader/ProductsHeader.tsx:21]: this is another example of style in element

File: src/components/LocationModal/LocationModal.tsx [hunk 3]
```diff
@@ -35,18 +35,30 @@ function LocationModal({ isOpen, onClose }: IPropsLocationModal): JSX.Element {
         isOpen={isOpen}
         onClose={onClose}>
         <ModalOverlay />
-        <ModalContent maxWidth={'47rem'} p={3} pt={10}>
+        <ModalContent maxWidth={{ base: '19rem', lg: '47rem' }} p={3} pt={10}>
```
Comment (dewale005) [inline src/components/LocationModal/LocationModal.tsx:38]: this is another example of style in element

File: src/components/LocationModal/LocationModal.tsx [hunk 4]
```diff
@@ -35,18 +35,30 @@ function LocationModal({ isOpen, onClose }: IPropsLocationModal): JSX.Element {
...
-          <ModalCloseButton size={'lg'} top={'2rem'} right={'3rem'} />
+          <ModalCloseButton
+            size={'lg'}
+            top={{ base: '1.5rem', lg: '2rem' }}
+            right={{ base: '1.5rem', lg: '3rem' }}
+          />
           <ModalBody pb={6}>
-            <Text fontSize={'20px'} mt={'1.5rem'} mb={'2.5rem'}>
+            <Text
+              fontSize={{ base: '10px', lg: '20px' }}
```
Comment (dewale005) [inline src/components/LocationModal/LocationModal.tsx:58]: this is another example of style in element

File: src/components/LocationModal/LocationModal.tsx [hunk 5]
```diff
@@ -35,18 +35,30 @@ function LocationModal({ isOpen, onClose }: IPropsLocationModal): JSX.Element {
...
+                fontSize: { base: '13px', lg: '25px' },
+                lineHeight: { base: '20px' },
+                mt: { base: '0.5rem', lg: '1rem' },
+              }}
             />
           </ModalHeader>
-          <ModalCloseButton size={'lg'} top={'2rem'} right={'3rem'} />
+          <ModalCloseButton
+            size={'lg'}
+            top={{ base: '1.5rem', lg: '2rem' }}
```
Comment (dewale005) [inline src/components/LocationModal/LocationModal.tsx:53]: this is another example of style in element

File: src/components/LocationModal/LocationModal.tsx [hunk 6]
```diff
@@ -35,18 +35,30 @@ function LocationModal({ isOpen, onClose }: IPropsLocationModal): JSX.Element {
...
-        <ModalContent maxWidth={'47rem'} p={3} pt={10}>
+        <ModalContent maxWidth={{ base: '19rem', lg: '47rem' }} p={3} pt={10}>
           <ModalHeader>
             <HeaderWithText
               heading="Set your location"
               text="See the lowest prices from pharmacies closest to you."
               headingStyle={{ sx: commonStyles.font32, variant: '' }}
-              textStyle={{ fontSize: '25px', mt: '1rem' }}
+              textStyle={{
+                fontSize: { base: '13px', lg: '25px' },
```
Comment (dewale005) [inline src/components/LocationModal/LocationModal.tsx:45]: this is another example of style in element

File: src/components/LocationModal/LocationModal.tsx [hunk 7]
```diff
@@ -56,43 +68,28 @@ function LocationModal({ isOpen, onClose }: IPropsLocationModal): JSX.Element {
...
                 />
               </InputGroup>
             </FormControl>
 
             <Flex
               alignItems={'center'}
               justifyContent={'space-between'}
-              my={'10'}>
+              my={{ base: '6', lg: '10' }}>
               <Divider />
```
Comment (dewale005) [inline src/components/LocationModal/LocationModal.tsx:80]: this is another example of style in element

**Review by dewale005 — CHANGES_REQUESTED** [review]
please address the styles done on the elements

**Review by dewale005 — APPROVED** [review]

**Pull request merged**